Sources/Features/Attributes/DefaultAttributes.swift

```diff
@@ -199,10 +199,19 @@ struct NetworkInfo: AttributesSource {
 struct LocationInfo: AttributesSource {
 
     var mutable: [String: Any?] {
-        return [
-            "location.enabled": CLLocationManager.locationServicesEnabled(),
-            "location.authorization.status": CLLocationManager.authorizationStatus().name
-        ]
+        var mutableDict = [String: Any?]()
```

Comment:
I think it won't work. The location information won't be returned on line 214 but will be generated later.

Reply:
seemed to work when I tested, I'll double check.